### DISCUSSION TITLE
Possible to merge people groups into one person?

### DISCUSSION THREAD

**Discussion by bclinton1** [photoview/photoview#335] [Ideas]
Hi folks! Just installed photoview and it looks really great. I am coming from Picasa and liked the way picasa managed the face recognition groups it created when it scanned images. I was able to group different people groups into one person. In other words photos of my son over his lifetime all into one single group for him. Right now photoview has several people groups for him as his face changed a lot of the years. Any chance of photoview incorporating something similar?

**Comment by viktorstrate** [comment]
First of all may Picasa rest in peace :)

Photoview actually does this already, though the settings might be a little hidden if you don't know they are there.
You can merge face groups by clicking the small gear next to the face name.

<img width="338" alt="Screenshot 2021-04-20 at 17 08 07" src="https://user-images.githubusercontent.com/4233458/115419882-fb99f900-a1fa-11eb-9ba8-288310c877dc.png">

**Reply by bclinton1** [reply]
Hmm.....I do not see that option with my container. That is exactly what was looking for. I am running this container on unraid. https://hub.docker.com/r/viktorstrate/photoview and it does not show that option.

**Reply by bclinton1** [reply]
Never mind...I found it. I was looking for it on the wrong page. I'm stupid :)

Thanks!
